src/ansys/mechanical/core/embedding/app.py

```diff
@@ -192,6 +205,10 @@ def __init__(self, db_file=None, private_appdata=False, **kwargs):
         self._updated_scopes: typing.List[typing.Dict[str, typing.Any]] = []
         self._subscribe()
 
+        globals = kwargs.get("globals")
+        if globals is not None and is_initialized():
```

Comment:
I don't think the `is_initialized` check is necessary here.  `initializer.initialize` already ran here (see line 187)